### PR TITLE
fix a number of NPE issues flagged by sonar

### PR DESCRIPTION
**Pull request by RangerRick**
Assuming these pass fine, they should be cherry-picked back to 32 or perhaps even `foundation-2023`.

### CODE REVIEW

File: opennms-webapp/src/main/java/org/opennms/web/admin/discovery/ActionDiscoveryServlet.java [hunk 1]
```diff
@@ -326,6 +326,10 @@
                 LOG.error(me.getMessage());
             }
 
+            if (proxy == null) {
+                throw new IllegalStateException("unable to get event proxy");
```
[the remark this answers is not in the document]
Comment (RangerRick) [reply opennms-webapp/src/main/java/org/opennms/web/admin/discovery/ActionDiscoveryServlet.java:330]: lol, gotta fix my fix

File: .circleci/build-triggers.override.json
Comment (mershad-manesh) [inline .circleci/build-triggers.override.json:1]: I'm not sure if this PR is ready to be delivered :). If it is, we might want to remove this file

Comment (RangerRick) [reply .circleci/build-triggers.override.json:1]: dangit! always forget

**Review by christianpape — APPROVED** [review]
LGTM.

**Pull request merged**